selfdrive/car/hyundai/values.py

```diff
@@ -127,8 +127,8 @@ class Buttons:
   {
     66: 8, 67: 8, 68: 8, 127: 8, 128: 8, 129: 8, 273: 8, 274: 8, 275: 8, 339: 8, 354: 3, 356: 4, 399: 8, 512: 6, 544: 8, 593: 8, 608: 8, 688: 5, 790: 8, 809: 8, 832: 8, 897: 8, 899: 8, 902: 8, 903: 8, 905: 8, 909: 8, 916: 8, 1040: 8, 1056: 8, 1057: 8, 1078: 4, 1151: 6, 1168: 7, 1170: 8, 1265: 4, 1280: 1, 1282: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1349: 8, 1351: 8, 1353: 8, 1356: 8, 1363: 8, 1366: 8, 1367: 8, 1369: 8, 1407: 8, 1414: 3, 1419: 8, 1427: 6, 1440: 8, 1456: 4, 1470: 8, 1486: 8, 1487: 8, 1491: 8, 1960: 8, 1990: 8, 1998: 8, 2000: 8, 2001: 8, 2004: 8, 2005: 8, 2008: 8, 2009: 8, 2012: 8, 2013: 8, 2015: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8
   }],
-  CAR.KIA_SPORTAGE: [{
-    67: 8, 68: 8, 127: 8, 273: 8, 274: 8, 275: 8, 339: 8, 356: 4, 399: 8, 447: 8, 512: 6, 544: 8, 593: 8, 608: 8, 688: 5, 790: 8, 809: 8, 832: 8, 884: 8, 897: 8, 899: 8, 902: 8, 903: 6, 909: 8, 916: 8, 1040: 8, 1078: 4, 1170: 8, 1191: 2, 1253: 8, 1254: 8, 1255: 8, 1265: 4, 1280: 1, 1282: 4, 1287: 4, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1349: 8, 1351: 8, 1353: 8, 1363: 8, 1365: 8, 1366: 8, 1367: 8, 1369: 8, 1407: 8, 1419: 8, 1427: 6, 1440: 8, 1456: 4, 1470: 8, 1472: 8, 1486: 8, 1487: 8, 1491: 8, 1492: 8, 1530: 8
+  CAR.KIA_SPORTAGE: [{   # Same as KIA OPTIMA SX 2019
```

Comment:
what is the thing that same as KIA OPTIMA SX 2019? fingerprint? model? config?

Reply:
The fingerprint for the  KIA SPORTAGE S 2020 is the same as the Fingerprint for the KIA OPTIMA SX 2019

Reply:
The value I provided was not being recognized. then I looked at the logs and saw that the system was using the KIA OPTIMA fingerprint for the SPORTAGE

Reply:
Only problem is how do we separate them so that we can use a specific tuning parameters for the SPORTAGE? As it is now it uses the tuning parameters for the OPTIMA

Reply:
that is problematic, Does Openpilot engage and steer properly? apart from tuning parameters

Reply:
It engages and steers fine. See attachment- near the bottom the platform used was The one for the optima. Then I forked it and had to delete the OPTIMA so that it would recognize the SPORTAGE...
![Capture](https://user-images.githubusercontent.com/60990482/76879567-cf1fea80-684c-11ea-97a1-158ccec52da1.PNG)

Reply:
...and I also had to change all references in the code for CAR.KIA_OPTIMA to CAR.KIA_SPOTRAGE for the comma two to recognize the car

Reply:
So, somehow the <car fingerprint> needs to be decoupled from the tuning settings.

Reply:
fixing yours this way will cause issue for others. we have to find a way to differentiate between them.
maybe we have to wait until VIN fingerprinting is working

Reply:
Yeah. Maybe so.